src/components/Calculator.js

```diff
@@ -4,6 +4,7 @@ import './Calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
 // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿